client/components/app-promo/lib/promo-retriever.js

```diff
@@ -0,0 +1,34 @@
+import { sample } from 'lodash';
+
+const promo_options = [
```

Comment:
Variables should be named with camelCase ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions)). Only reason the camel case ESLint rule is not active is because working with the REST API often requires creating request objects with snake-case properties.